### ISSUE TITLE
Using node breakers with masq causes player who placed it to starve

### ISSUE THREAD

**Issue by mtango688** [minetest-mods/pipeworks#146]
Having masquerade_as_owner set to true in wielder causes your hunger bar to deplete far more rapidly than it should if you use a hunger/stamina mod.

In my experience that value should always be set to false.

**Comment by Ekdohibs** [comment]
No, setting that value to false causes problems with protection mods in particular: you wouldn't be able to use node breakers/deployers inside protected areas. What's more, they could be exploited with other mods such as minegeld.

**Comment by mtango688** [comment]
Great. Meanwhile, if you place a node breaker in public areas and someone else uses it you will starve. How does that make any sense?

**Comment by t4im** [comment]
This sounds like a something to be addressed in the hunger mod.
It could easily test for a player being real or not before starving you.
Just open an issue with them and ask to check for [is_fake_player](
https://github.com/minetest-mods/pipeworks/blob/6de437b83d99cdc49dc7899b3bf4fc6f94054d92/wielder.lua#L84)
